### PR TITLE
feat: add translations support

### PR DESCRIPTION
**Pull request by BryanttV**
### Description
This PR adds support for translations in XBlock. Added translations for `es_419` and `es_ES`

### How To Test
1. Install this branch in your environment.
2. Change the platform language in `(Account → Site Preferences → Site Language)`. Set it to **Spanish (Latin America)**
3. Add a block in a unit from Studio.
4. You should see the block configuration in the selected language.

### Screenshots
image
image

### CODE REVIEW

File: controlled_navigation/locale/es_419/LC_MESSAGES/text.po [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+
+msgid ""
+"Text that will be displayed on the previous button to navigate to the "
+"previous child."
+msgstr ""
+"Texto que se mostrará en el botón anterior para navegar al hijo anterior."
+
+#: controlled_navigation.py:56
+msgid "Previous Question"
+msgstr "Anterior Pregunta"
```
Comment (mariajgrimaldi) [inline controlled_navigation/locale/es_419/LC_MESSAGES/text.po]:
```suggestion
msgstr "Pregunta Anterior"
```

File: controlled_navigation/locale/es_ES/LC_MESSAGES/text.po [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+
+msgid ""
+"Text that will be displayed on the previous button to navigate to the "
+"previous child."
+msgstr ""
+"Texto que se mostrará en el botón anterior para navegar al hijo anterior."
+
+#: controlled_navigation.py:56
+msgid "Previous Question"
+msgstr "Anterior Pregunta"
```
Comment (mariajgrimaldi) [inline controlled_navigation/locale/es_ES/LC_MESSAGES/text.po]: Same suggestion here

**Review by mariajgrimaldi — APPROVED** [review]
LGTM! Just a few comments left to address. Thank you!

File: controlled_navigation/locale/es_ES/LC_MESSAGES/text.po (hunk 2 above)
Comment (BryanttV) [reply controlled_navigation/locale/es_ES/LC_MESSAGES/text.po]: Where?

File: controlled_navigation/locale/es_419/LC_MESSAGES/text.po (hunk 1 above)
Comment (BryanttV) [reply controlled_navigation/locale/es_419/LC_MESSAGES/text.po]: Done: https://github.com/eduNEXT/xblock-controlled-navigation/commit/656944eb437a189e308105d11a5d7e26a60094a2

**Pull request merged**